stream-testkit/src/test/scala/org/apache/pekko/stream/testkit/StreamTestKitSpec.scala

```diff
@@ -136,15 +136,15 @@ class StreamTestKitSpec extends PekkoSpec {
         // It also needs to be dilated since the testkit will dilate the timeout
         // accordingly to `-Dpekko.test.timefactor` value.
         val initialDelay = (timeout * 2).dilated
+        val pf: PartialFunction[Any, Unit] = {
```

Comment:
these changes seem quite small compared to the StreamTestKit changes

is it possible to test all the new functions?

Reply:
Sorry, I'm a little busy at $Work right now, testing all these new methods seems like a lot of work, and as we can see, the changes are mostly small, just forwarding/converting.